oa2-lists/asset_network.csv

```diff
@@ -0,0 +1,95 @@
+OpenAlias asset_network,Asset Common Name,CAIP-2 Prefix
+ada,Cardano,
+aleo,Aleo Network,aleo:0
+algo,Algorand,algorand:wGHE2Pwdvd7S12BL5FaOP20EGYesN73k
+aptos,Aptos,
+arb,Arbitrum One,eip155:42161
+arbn,Arbitrum Nova,eip155:42170
+ark,ARK,
+arweave,Arweave,arweave:7wIU
+avaxc,Avalanche C-Chain,avax:8aDU0Kqh-5d23op-B-r-4YbQFRbsgF9a
+avaxp,Avalanche P-Chain,avax:Rr9hnPVPxuUvrdCul-vjEsU1zmqKqRDo
+avaxx,Avalanche X-Chain,avax:imji8papUf2EhV3le337w1vgFauqkJg-
+ban,Banano,
+base,Base,eip155:8453
+bch,Bitcoin Cash,bip122:
```

Comment:
[BIP-122](https://github.com/bitcoin/bips/blob/master/bip-0122.mediawiki#definition-of-chain-id) says "for forked chains, it's the block hash of the first block after fork".
According to WIkipedia, split block is 478559. I guess that means Bitcoin Cash is bip122:000000000000000000651ef99cb9fcbe

https://blockchair.com/bitcoin-cash/block/478559

Reply:
Or, if "first block after fork" is a first block after the split point: bip122:000000000000000000b15ad892af8f6a

https://blockchair.com/bitcoin-cash/block/478560